Front-end/Auction_App_Front/app/components/last-chance.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+ httpBase: service(),
+ lastchance: null,
+ pageSize: '8',
+ init() {
+  this._super(...arguments);
+  this.get('httpBase').ajaxReq('http://localhost:8080/api/lastchance', this.get('pageSize'), 'POST').then((result) => {
```

Comment:
This should be moved to a seperate service, also don't hardcode local host, it should be a host variable

Reply:
Same for all requests below